transforms/universal/pdf2parquet/ray/Dockerfile

```diff
@@ -0,0 +1,50 @@
+ARG BASE_IMAGE=docker.io/rayproject/ray:2.24.0-py310
+
+FROM ${BASE_IMAGE}
+
+RUN \
+    sudo apt-get update \
+    # for opencv, towhee
+    && sudo apt-get install -y libgl1 libglib2.0-0 curl wget \
+    && sudo apt-get clean
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=ray:users data-processing-lib-python/ data-processing-lib-python/
+RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
+COPY --chown=ray:users data-processing-lib-ray/ data-processing-lib-ray/ 
+RUN cd data-processing-lib-ray    && pip install --no-cache-dir -e .
+COPY --chown=ray:users python-transform/  python-transform/
+RUN cd python-transform && pip install --no-cache-dir -e .
+
+
+COPY --chown=ray:users pyproject.toml pyproject.toml 
+RUN pip install --no-cache-dir -e .
+
+# Download models
+RUN python -c 'from deepsearch_glm.utils.load_pretrained_models import load_pretrained_nlp_models; load_pretrained_nlp_models(verbose=True);'
+# RUN python -c 'from docling.document_converter import DocumentConverter; from pathlib import Path; DocumentConverter.download_models_hf(local_dir=Path("./artifacts/"));'
+RUN python -c 'from docling.document_converter import DocumentConverter; s=DocumentConverter.download_models_hf(); print(f"Models cached in {s}")'
+
+# copy source data
+COPY --chown=ray:users src/ .
```

Comment:
You need to copy in pdf2parquet_transform_ray.py to be in the home dir.

Reply:
this is what this command is doing, or am I missing something?

Reply:
see https://github.com/dolfim-ibm/data-prep-kit/blob/644a8766d896e2dec802b6bb43a7ab5eab4c5eab/transforms/universal/pdf2parquet/kfp_ray/pdf2parquet_wf.py#L23, for example.